pkg/platform/kube/apigateway.go

```diff
@@ -0,0 +1,38 @@
+package kube
+
+import (
+	"github.com/nuclio/nuclio/pkg/platform"
+
+	"github.com/nuclio/errors"
+)
+
+func ValidateUpstreamSpec(apiGatewaySpec *platform.APIGatewaySpec) error {
+	upstreams := apiGatewaySpec.Upstreams
+
+	if len(upstreams) > 2 {
+		return errors.New("Received more than 2 upstreams. Currently not supported")
+	} else if len(upstreams) == 0 {
+		return errors.New("One or more upstreams must be provided in spec")
+	} else if apiGatewaySpec.Host == "" {
```

Comment:
Suggest we make this a bit looser and enrich "" -> "/" instead of failling.
Any reason not to? WDYT?

Reply:
a _host_ must be provided, you may got it confused with _path_

Reply:
yes true dat. was thinking of path